src/psyclone/psyir/nodes/node.py

```diff
@@ -340,6 +341,7 @@ def dag_gen(self, graph):
             graph.edge(remote_name, local_name, color="blue")
         # now call any children so they can add their information to
         # the graph
+        from psyclone.psyir.nodes import Loop
```

Comment:
pylint complains that Loop is being re-imported.

Reply:
Indeed it is, same merging problem.